### PR TITLE
Adapt the Javadoc to look like the golodoc one

### PR DESCRIPTION
**Pull request by yloiseau**
- add a link to source
- change style to match the golodoc style
- add highlight support in the generated javadoc

### CODE REVIEW

File: doc/stylesheet.css [hunk 1]
```diff
@@ -0,0 +1,249 @@
+/* Javadoc style sheet */
+/*
+Overall document style
```
Comment (jponge) [inline doc/stylesheet.css]: /*

Comment (yloiseau) [inline doc/stylesheet.css]: can you be more precise? :smile:

Comment (jponge) [inline doc/stylesheet.css]: you lack one on this line :-)

**Pull request merged**